storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/integration/GetOperationTest.java

```diff
@@ -321,6 +401,58 @@ public void getNonExistedObjectExecuteAsObservable() {
     }
 
     @Test
+    public void getExistedObjectExecuteAsSingle() {
+        TestItem expectedItem = TestItem.create(null, "value");
+        contentResolver.insert(TestItem.CONTENT_URI, expectedItem.toContentValues());
+        contentResolver.insert(TestItem.CONTENT_URI, TestItem.create(null, "value1").toContentValues());
+        contentResolver.insert(TestItem.CONTENT_URI, TestItem.create(null, "value2").toContentValues());
+
+        Single<TestItem> testItemSingle = storIOContentResolver
+                .get()
+                .object(TestItem.class)
+                .withQuery(Query.builder()
+                        .uri(TestItem.CONTENT_URI)
+                        .where(TestItem.COLUMN_VALUE + "=?")
+                        .whereArgs("value")
+                        .build())
+                .prepare()
+                .asRxSingle();
+
+        TestSubscriber<TestItem> testSubscriber = new TestSubscriber<TestItem>();
+        testItemSingle.subscribe(testSubscriber);
+
+        testSubscriber.awaitTerminalEvent(5, SECONDS);
+        testSubscriber.assertNoErrors();
+
+        List<TestItem> emmitedItems = testSubscriber.getOnNextEvents();
+        assertThat(emmitedItems.size()).isEqualTo(1);
+        assertThat(expectedItem.equalsWithoutId(emmitedItems.get(0))).isTrue();
+    }
```

Comment:
Please add `changesTestSubscriber`, test after this may see notification from previous test…